docs/002-basics-of-ci-with-actions.md

```diff
@@ -2,48 +2,48 @@
+This repository contains a [React](https://reactjs.org/)-based application built with [Vite](https://vitejs.dev/). We aim to automate its testing and building in this segment of the workshop.
 
-You can browse through the files if you are interested in how the app itself works (this is not strictly necessary to understand the rest of the workshop though).
+Feel free to explore the files if you're curious about the app's operation (though it's not strictly necessary for understanding the remainder of the workshop).
 
-- [`src/main.tsx`](../src/main.ts) : is the main entry point of the application
-- [`src/pages/Home.tsx`](../src/pages/Home.tsx) : is the route that contains most of what you see when starting the application
-- [`src/pages/Home.test.ts`](../src/pages/Home.test.tsx) : contains [`vitest`](https://vitest.dev/) tests which we will run with GitHub Actions
-- [`Dockerfile`](../Dockerfile) : a Docker file that package the application in a container for release in a later step.
+- [`src/main.tsx`](../src/main.ts) : This is the main entry point of the application.
+- [`src/pages/Home.tsx`](../src/pages/Home.tsx) : This route contains most of what you will see upon launching the application.
+- [`src/pages/Home.test.ts`](../src/pages/Home.test.tsx) : Here, you will find [`vitest`](https://vitest.dev/) tests that we will run with GitHub Actions.
+- [`Dockerfile`](../Dockerfile) : This Docker file packages the application into a container that will be used later in this workshop.
 
-If if you want to test the application, you can start a Codespaces and run it (`npm run dev`) or test it (`npm test`). If you want to run the application on your local machine, you will need to install Node.js.
+If you want to test the application, you can start a Codespace and run it with the command `npm run dev`
+(you can also run the tests with `npm test`). To run the application on your local machine, you will need to install Node.js first.
 
-To test the container, run `docker build . -t local:latest` to build the image and then `docker run -p 8080:8080 local:latest` to run it. Running these commands locally will require you to install [Docker](https://www.docker.com/).
+To test the container, run `docker build . -t local:latest` to build the image and `docker run -p 8080:8080 local:latest` to run it. These commands require a local installation of [Docker](https://www.docker.com/).
 
-## 2 - Add Continuous Integration
+## 2 - Implement Continuous Integration (CI)
 
 ### 2.1 - Use a starter workflow
 
-To develop a GitHub Workflow process that uses Actions to automate the Continuous Integration process, you can begin by adding a **starter workflow** to the repository:
+To build a workflow that employs Actions for automating the Continuous Integration process, start by adding a **starter workflow** to your repository:
```

Comment:
```suggestion
To build a workflow that employs Actions for your Continuous Integration process, start by adding a **starter workflow** to your repository:
```

The phrase `automating the CI process` doesn't really make sense to me, as CI is something that cant' be really done manually, can it?